src/pages/ReimbursementAccount/BusinessInfo/substeps/ConfirmationBusiness.tsx

```diff
@@ -0,0 +1,182 @@
+                    title={values[businessInfoStepKeys.COMPANY_WEBSITE]}
+                    shouldShowRightIcon
+                    onPress={() => {
+                        onMove(2);
+                    }}
+                />
+                <MenuItemWithTopDescription
+                    description={translate('businessInfoStep.companyType')}
+                    title={values[businessInfoStepKeys.INCORPORATION_TYPE]}
+                    shouldShowRightIcon
+                    onPress={() => {
+                        onMove(5);
+                    }}
+                />
+                <MenuItemWithTopDescription
+                    description={translate('businessInfoStep.incorporationDate')}
+                    title={values[businessInfoStepKeys.INCORPORATION_DATE]}
+                    shouldShowRightIcon
+                    onPress={() => {
+                        onMove(6);
+                    }}
+                />
+                <MenuItemWithTopDescription
+                    description={translate('businessInfoStep.incorporationState')}
+                    title={translate(`allStates.${values[businessInfoStepKeys.INCORPORATION_STATE] as States}.stateName`)}
+                    shouldShowRightIcon
+                    onPress={() => {
+                        onMove(7);
+                    }}
+                />
+                {/* @ts-expect-error TODO: Remove this once FormProvider (https://github.com/Expensify/App/issues/31972) is migrated to TypeScript */}
+                <FormProvider
+                    formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
+                    validate={validate}
+                    onSubmit={onNext}
+                    scrollContextEnabled
+                    submitButtonText={translate('common.confirm')}
+                    style={[styles.mh5, styles.flexGrow1]}
+                >
+                    <CheckboxWithLabel
+                        aria-label={`${translate('businessInfoStep.confirmCompanyIsNot')} ${translate('businessInfoStep.listOfRestrictedBusinesses')}`}
+                        inputID={businessInfoStepKeys.HAS_NO_CONNECTION_TO_CANNABIS}
+                        defaultValue={defaultCheckboxState}
+                        LabelComponent={() => (
+                            <Text>
+                                {`${translate('businessInfoStep.confirmCompanyIsNot')} `}
+                                <TextLink
+                                    // eslint-disable-next-line max-len
+                                    href={CONST.LIST_OF_RESTRICTED_BUSINESSES}
+                                >
+                                    {`${translate('businessInfoStep.listOfRestrictedBusinesses')}.`}
+                                </TextLink>
+                            </Text>
+                        )}
+                        style={[styles.mt4]}
+                        shouldSaveDraft
+                        onInputChange={() => {}}
```

Comment:
I think it is not needed - onInputChange is not required and is `()=>{}` by default